satellite/console/consoleweb/consoleql/project.go

```diff
@@ -189,6 +201,35 @@ func graphqlProject(service *console.Service, types *TypeCreator) *graphql.Objec
 					return service.GetBucketTotals(p.Context, project.ID, cursor, before)
 				},
 			},
+			FieldPaymentMethods: &graphql.Field{
+				Type: graphql.NewList(types.paymentMethod),
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					project, _ := p.Source.(*console.Project)
+
+					paymentMethods, err := service.GetProjectPaymentMethods(p.Context, project.ID)
```

Comment:
I think PaymentOptions is more appropriate here, rather than Methods (which in my definition would be Card, bar, etc.)

Reply:
Options in a sense of choosing one of the CC cards.